### DISCUSSION TITLE
Poll devices on startup?

### DISCUSSION THREAD

**Discussion by kris10an** [zwave-js/zwave-js-ui#2050] [Q&A] [answered]
Hi,

Just wondering if there is some way to poll devices on startup/boot?
I'm trying to reduce traffic by configuring devices to send less frequent reports and instead report on changes in for example power consumption. And still get correct consumption data. So far so good, but I noticed some inconsistencies during a period I was doing several reboots. An example:
![Water boiler](https://user-images.githubusercontent.com/17658084/145814699-347ad8c6-7701-4bb6-b699-ac6e315cfd70.PNG)

This is my water boiler power (W) and total energy consumption (kWh). It updates quite well until I did some reboots (highlighted in yellow). The device (Aeotec Heavy duty switch if relevant) probably updated its power value while the server was down. Is there any setting/configuration to make sure values are refreshed/polled on startup?
I could make an automation in Home Assistant, but if there is a better way I'd prefer that :)

**Comment by robertsLando** [comment, marked as answer]
Are you using zwavejs integration?

**Reply by kris10an** [reply]
Yes

**Reply by robertsLando** [reply]
@kris10an If so you should ask in hass core repo, I don't handle what is sent to hass I just start the zwavejs-server that is not maintained by me and is used by hass to get the nodes and events from zwavejs using websockets
